muxers/yamux/src/lib.rs

```diff
@@ -386,23 +379,16 @@ where
     }
 }
 
-/// The Yamux [`StreamMuxer`] error type.
-#[derive(Debug, Error)]
-#[error("yamux error: {0}")]
-pub struct YamuxError(#[from] yamux::ConnectionError);
-
-impl From<YamuxError> for io::Error {
-    fn from(err: YamuxError) -> Self {
-        match err.0 {
-            yamux::ConnectionError::Io(e) => e,
-            e => io::Error::new(io::ErrorKind::Other, e),
-        }
+fn to_io_error(e: yamux::ConnectionError) -> io::Error {
```

Comment:
```suggestion
fn into_io_error(e: yamux::ConnectionError) -> io::Error {
```
https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
Good thinking! Will include in https://github.com/libp2p/rust-yamux/pull/135.

Reply:
Done: https://github.com/libp2p/rust-yamux/compare/b4ddd2afd6215081b952512ceba27eb45987c1e0..fb6d9ab0ea3406c87dbd076601790e5b4c85e8d3